Marlin/src/gcode/feature/ft_motion/M493.cpp

```diff
@@ -155,15 +156,12 @@ void GcodeSuite::M493() {
 
   if (!parser.seen_any())
     flag.report_h = true;
-  else
-    planner.synchronize();
```

Comment:
Is it safe to change the FT Motion system mode and/or parameters while a move is still in progress in the planner? That is why the synchronize is done at the top. We'll need to test whether arbitrary calls to `M493` during a print job cause any trouble.